sklearn/mixture/gmm.py

```diff
@@ -429,6 +450,7 @@ def fit(self, X, y=None):
         """
         # initialization step
         X = check_array(X, dtype=np.float64)
+        self.responsibilities_ = None
```

Comment:
The attributes with a trailing `_` should only be updated inside the `fit` method (or the `partial_fit` method). The `predict_proba` method should not mutate the internal state of the estimator.